change/@ni-nimble-angular-4fac20cb-60b2-4732-b697-bb71f0a26238.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
All these can be `patch` since it's an incubating component.

But you will need to fix the breaking change in systemlink-lib-angular since the sl-table columns expose these.